crates/biome_js_analyze/src/lint/style/use_self_closing_elements.rs

```diff
@@ -53,6 +57,25 @@ declare_lint_rule! {
     /// ```js
     /// <Foo.bar>child</Foo.bar>
     ///```
+    ///
+    /// ## Options
+    ///
+    /// Default: true
+    ///
+    /// ```json
+    /// {
+    ///     "//":"...",
+    ///     "options": {
+    ///         "html": true
```

Comment:
IMHO, the name `html` is not descriptive enough. A name should be explanatory enough so users can understand what it does just by reading it.

I think we should name it `checkHtmlElements`, what do you think? 

Regarding the structure of the docs, it should be like this: 

``````md
### Options

#### `nameOfTheOoption`

Default: `true`

<Explanation of the option> (e.g. This option allows to control whether this rule should check HTML elements or not)

<Explanation of an example> (e.g. In the following example, when the option is set to `false`, this happens)

```json
{
	"//": "example of the configuration"
}
```

```jsx,ignore
// example of code that will pass, use the `ignore` directive in the code block, so the codegen won't execute the code
<img src="something.png" >
```

``````

Reply:
Thanks for the feedback. I think we can proceed with `checkHtmlElements`!

Reply:
Addressed in https://github.com/biomejs/biome/pull/3697/commits/c633de84c42f3c894d0960277be0d216b7bbc459

Please let me know if it's acceptable.

Reply:
@Conaclos I ended up using your suggestion!